hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocolPB/DatanodeProtocolClientSideTranslatorPB.java

```diff
@@ -188,8 +188,7 @@ public HeartbeatResponse sendHeartbeat(DatanodeRegistration registration,
 
   @Override
   public DatanodeCommand blockReport(DatanodeRegistration registration,
-      String poolId, StorageBlockReport[] reports,
-      BlockReportContext context)
+      String poolId, StorageBlockReport[] reports, BlockReportContext context)
```

Comment:
NIT: unnecessary formatting change can be avoided.

Reply:
already fixed